packages/plugins/EVM/src/state/IdentityService.ts

```diff
@@ -24,15 +24,16 @@ const RSS3_URL_RE = /https?:\/\/(?<name>[\w.]+)\.(rss3|cheers)\.bio/
 const RSS3_RNS_RE = /(?<name>[\w.]+)\.rss3/
 const LENS_RE = /[^\s()[\]]{1,256}\.lens\b/i
 const LENS_URL_RE = /https?:\/\/.+\/(\w+\.lens)/
+const LENS_DOMAIN_RE = /\w+.lens/
```

Comment:
```suggestion
const LENS_DOMAIN_RE = /\w+\.lens/
```

Reply:
Please find out the accurate rule of profile handles in the Lens protocol codebase or documentation and attach a comment here.

Reply:
https://github.com/lens-protocol/lens-sdk/blob/main/packages/api-bindings/src/graphql/utils/isValidHandle.ts